imports/collections/schemas/tags.js

```diff
@@ -16,7 +16,7 @@ import { Metafield } from "./metafield";
  * @property {String[]} relatedTagIds optional
  * @property {Boolean} isDeleted default value: `false`
  * @property {Boolean} isTopLevel required
- * @property {Boolean} isVisible defalut value: `true`
```

Comment:
nice catch

Reply:
Code Spell Checker extension caught it, but I'll take credit. 😄